Automattic/newspack-plugin
# feat: disable GA4 front end custom params by default

### All Submissions:






### Changes proposed in this Pull Request:



Adding custom parameters had an unintended consequence in some sites. Some params went missing from default events.

While we investigate how to fix it we will disable this by default.

### How to test the changes in this Pull Request:

1. On `trunk`, visit a site with Site kit configured.
2. See the page source and look for `gtag("config"`
3. Confirm you see the params there, added as the third argument of this function: `{"logged_in":"yes",...`
4. Checkout this branch and reload
5. Confirm the custom params are gone
6. Edit your wp-config.php and add `define( 'NEWSPACK_GA_ENABLE_CUSTOM_FE_PARAMS', true );`
7. Reload and confirm the custom params are back

### Other information:
